alpha/lib/model/category.php

```diff
@@ -591,7 +602,21 @@ protected function addCopyCategoryKuserJob($categoryId)
 
 		kJobsManager::addCopyJob($this->getPartnerId(), CopyObjectType::CATEGORY_USER, $filter, $templateCategory);
 	}
-	
+
+	protected function addDeleteCategoryTreeEntryJob($categoryId, $fullIds)
+	{
+		$filter = new categoryEntryFilter();
+		$filter->setFullIdsStartsWith($fullIds);
+
+		$c = new Criteria();
+		$c->add(categoryEntryPeer::CATEGORY_ID, $categoryId);
+		if(!categoryEntryPeer::doSelectOne($c)) {
```

Comment:
This test is wrong, you may have category-entry objects that associated with one of the category descendants